config.analysis

```diff
@@ -41,6 +48,9 @@ climo_yr1 = 6
 climo_yr2 = 10
 # the offset year to be added to simulation years
 yr_offset = 1849
+# start and end date for the analysis
+analysis_start_date = 0001-01-01_00:00:00
+analysis_end_date = 9999-01-01_00:00:00
```

Comment:
Got it.  So basically either way we need to make sure we have some buffer where we search for files outside the specified range.  I guess the buffer has to come from the file writing interval.  I'll see what can be done, but it should be possible to try reading one file too many in both directions to be sure we have all the correct dates, then `mpas_xarray` can sort out the details.